optional-container-engine/README.md

```diff
@@ -0,0 +1,73 @@
+# Deploy Bookshelf to Container Engine/Kubernetes
+
+This optional tutorial will walk you through how to deploy the Bookshelf sample application to [Google Container Engine](https://cloud.google.com/container-engine/). This tutorial is also applicable to [Kubernetes](http://kubernetes.io/) outside of Container Engine, but may require additional steps for external load balancing.
+
+## Pre-requisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com).
+
+2. [Enable billing](https://console.cloud.google.com/project/_/settings) for your project. 
+
+3. Install the [Google Cloud SDK](https://cloud.google.com/sdk)
+
+        $ curl https://sdk.cloud.google.com | bash 
+        $ gcloud init
+
+4. Install [Docker](https://www.docker.com/).
+
+## Create a cluster
+
+Create a cluster for the bookshelf application:
+
+    gcloud container clusters create bookshelf \
+        --scope "https://www.googleapis.com/auth/userinfo.email","cloud-platform"
+
+The scopes specified in the `--scope` argument allows nodes in the cluster to access Google Cloud Platform APIs, such as the Cloud Datastore API.
+
```

Comment:
They won't need it if it's their first time making a container engine app...but teaching people to use `gcloud container clusters get-credentials` can save them a lot of pain if they work with multiple clusters.